remoteappmanager/docker/container_manager.py

```diff
@@ -165,7 +158,7 @@ def _start_container(self, user_name, image_name, volumes):
         # build the dictionary of keyword arguments for create_container
         container_name = _generate_container_name("remoteexec",
                                                   user_name,
-                                                  image_name)
+                                                  mapping_id)
```

Comment:
To clarify:

**application**: The docker image to use 
**policy**: The container setup for connecting volumes and the execution constrains 

The mapping id is unique for each combination of (application, policy) tuple. The id is persistent across runs of the single user app and jupyter hub server.